src/client/components/Pagination/Pagination.component.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './Pagination.styles.css';
+
+// Floor and setFloor states will be passed to us as props from the parent container
+function Pagination({ floor, setFloor }) {
+  // Floor array is an array that doesn't change or mutate and it will always has the same values, the number of floors.
+  const floorArray = [0, 1, 2, 3, 4, 5];
```

Comment:
Shouldn't this suppose to be a prop as well, why is the page index being hardcoded? The parent component should know the total pages, right?

Reply:
Thanks for the review.
For the array , the parent components don't have it as a whole as they may need one each time but for my component I need to have it all as it might change between pages.

Reply:
I have worked on the parent component and as Dlnya says here, I actually do not use this array as I don't need to render all of them at a time like pagination. I passed one prop only, i.e one floor and that renders only.

Reply:
I see. thanks for the explanation. Since it the `floorArray` might change, should it be a variable?

Reply:
if I am not mistaken in understanding your point,
the array itself is fixed and its only those 6 numbers, what changes is the floor.  which I have mapped through .
and for the api , it is not external and that is why I was told not to use useEffect as  I had used before.
if your point is not using array , kindly explain me. Thank you

Reply:
No, its not about the data type you are declaring; it can be anything you find suitable for it and I like the idea you are using array. I am just wondering why are these six number were hardcoded. 

If `const floorArray = [0, 1, 2, 3, 4, 5]` will always be fixed, then feel free to merge.

Reply:
Now I understand , I thought you are recommending another way.
Yes , It is fixed and won't change.
thank you